.github/workflows/ci-build.yml

```diff
@@ -164,8 +168,21 @@ jobs:
         driver: docker-container
         use: true
     - name: Build
+      id: bake
       uses: docker/bake-action@v4.0.0
       with:
         files: docker-bake.hcl
         workdir: .
-        targets: interop_binaries
+        targets: interop_binaries_small
+        load: true
+        set: |
+          janus_interop_client_small.tags=janus_interop_client
+          janus_interop_aggregator_small.tags=janus_interop_aggregator
+          janus_interop_collector_small.tags=janus_interop_collector
+    - name: Save tarball
+      run: docker save --output=/tmp/images.tar janus_interop_client janus_interop_aggregator janus_interop_collector
+    - name: Upload artifact
+      uses: actions/upload-artifact@v3
```

Comment:
This will upload the three docker image, which are 40ish MB apiece, on every PR build. I'm concerned about our utilization of artifacts storage. The [GitHub docs](https://docs.github.com/en/actions/using-workflows/storing-workflow-data-as-artifacts) discuss a 90 day default retention period for artifacts, but nothing about a maximum on storage. Maybe we should go ahead with this and see if we start getting angry emails about using up our storage quota?

Or maybe we should preemptively tighten the retention period down to maybe 1 day, since we expect the artifacts emitted by this job to be immediately consumed by the test job.